eth/protocol.go

```diff
@@ -36,15 +47,37 @@ const (
 
 // eth protocol message codes
 const (
-	StatusMsg = iota
-	NewBlockHashesMsg
-	TxMsg
-	GetBlockHashesMsg
-	BlockHashesMsg
-	GetBlocksMsg
-	BlocksMsg
-	NewBlockMsg
-	GetBlockHashesFromNumberMsg
+	// Protocol messages belonging to eth/61
+	StatusMsg                   = 0x00
+	NewBlockHashesMsg           = 0x01
+	TxMsg                       = 0x02
+	GetBlockHashesMsg           = 0x03
+	BlockHashesMsg              = 0x04
+	GetBlocksMsg                = 0x05
+	BlocksMsg                   = 0x06
+	NewBlockMsg                 = 0x07
+	GetBlockHashesFromNumberMsg = 0x08
+
+	// Protocol messages belonging to eth/62 (new protocol from scratch)
+	// StatusMsg          = 0x00 (uncomment after eth/61 deprecation)
+	// NewBlockHashesMsg  = 0x01 (uncomment after eth/61 deprecation)
+	// TxMsg              = 0x02 (uncomment after eth/61 deprecation)
+	GetBlockHeadersMsg = 0x03
+	BlockHeadersMsg    = 0x04
+	GetBlockBodiesMsg  = 0x05
+	BlockBodiesMsg     = 0x06
+	// 	NewBlockMsg       = 0x07 (uncomment after eth/61 deprecation)
+
+	// Protocol messages belonging to eth/63
+	GetNodeDataMsg = 0x0d
+	NodeDataMsg    = 0x0e
+	GetReceiptsMsg = 0x0f
+	ReceiptsMsg    = 0x10
+
+	// Protocol messages belonging to eth/64
+	GetAcctProofMsg     = 0x11
+	GetStorageDataProof = 0x12
```

Comment:
Ah, good catch :). I just started adding these packages and then saw that there's quite a lot of database side logic missing so I never bothered to finish them up. Will correct nonetheless :)